src/app/modules/main-module/components/main/main.component.ts

```diff
@@ -110,35 +129,46 @@ export class MainComponent implements OnInit {
     if (this.word[this.positionInput] !== '') {
       return;
     }
-
     if (this.positionInput > this.word.length - 1 || this.positionInput < 0) {
       this.positionInput = this.word.length - 1;
-
       return;
     }
-
     if (this.positionInput > 0) {
       this.positionInput--;
-
       return;
     }
   }
 
   deleteLetter() {
     this.changePositionWhenDelete();
-
     this.word[this.positionInput] = '';
   }
 
   setValuesWord() {
     this.wordSend.pos0 = this.word[0];
-
     this.wordSend.pos1 = this.word[1];
-
     this.wordSend.pos2 = this.word[2];
-
     this.wordSend.pos3 = this.word[3];
-
     this.wordSend.pos4 = this.word[4];
   }
+
+  checkWin() {
+    let resultado = this.wordStatus.join(',');
+
+    if (resultado.toLocaleUpperCase() === this.winValue) {
+      this.openDialogWin(
+        'Enhorabuena has acertado la palabra, pero... ¿podrás con la siguiente?'
```

Comment:
los textos mejor meterlos en variables, tanto este como el del otro del button. Después otra duda, si openDialogWin solo se llama desde aquí, para qué tenéis que pasarle parámetros?